openquake/hazardlib/tests/calc/disagg_test.py

```diff
@@ -146,10 +146,7 @@ def test_no_filters(self):
         (mags, dists, lons, lats, trts, trt_bins, probs_no_exceed) = \
             disagg._collect_bins_data_old(
                 self.sources, self.site, self.imt, self.iml, self.gsims,
-                self.truncation_level, n_epsilons=3,
-                source_site_filter=filters.source_site_noop_filter,
-                rupture_site_filter=filters.rupture_site_noop_filter
-                )
+                self.truncation_level, n_epsilons=3)
```

Comment:
I removed a line too much. I am restoring it. This test is not sensitive to the filtering though.